includes/class-wsuwp-person-display.php

```diff
@@ -132,12 +127,148 @@ public function template_include( $template ) {
 	public function content() {
 		remove_filter( 'the_content', array( $this, 'content' ) );
 
+		$local_record_id = get_post()->ID;
+		$nid = get_post_meta( $local_record_id, '_wsuwp_profile_ad_nid', true );
+		$person = WSUWP_People_Post_Type::get_rest_data( $nid );
+		// Store _use_{*} metas as an array????
```

Comment:
Haha, I decided this would be too much of a pain for other reasons, then forgot to remove the comment.